scripts/extract-ios-device-names.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-env node */
-import fs from 'node:fs';
+import {existsSync, unlinkSync} from 'node:fs';
+import fs from 'node:fs/promises';
 import path from 'node:path';
-
 import prettier from 'prettier';
```

Comment:
does biome have anything we can use here instead?

Reply:
if it ends up being committed it would also get picked up by lint

Reply:
You're absolutely right. Removing prettier from the script.